docs/user-guide/projects/project-lists/README.md

```diff
@@ -75,7 +85,7 @@ Projects can be filtered in OpenProject. The default view will list all currentl
 
 To adjust the project lists view use the **Filters** button, select the filtering criteria you require and click the blue **Apply** button.
 
-If you want to save this filtered list use the menu in the far right corner (three dots) and click **Save**.
+If you want to save this filtered list use the menu in the far right corner (three dots) and click **Save as**. Alternatively you can also use **Save as link** next to the warning message.
```

Comment:
I would change the order of this sentence (and screenshots) and do some small changes:

"If you want to save this filtered list use the link **Save as** next to the information message in the page header or alternatively click on the menu (three dots) and click **Save as**.